### PR TITLE
8.0 account invoice line default account

### PR DESCRIPTION
**Pull request by jbaudoux**
Porting this MP from lp to github
https://code.launchpad.net/~bcim/account-invoicing/7.0-account_invoice_line_default_account

LP description: Extracted account_invoice_line_default_account module version 6.1 from account-financial-tools project and ported to 7.0. I'm pushing this migrated module to this account-invoicing project as this module is for invoicing and is not a financial tool

Ported to 8.0

### CODE REVIEW

**Comment by jgrandguillaume** [comment]
Hi !

Happy to see you over here :) Can you just have a look why Travis is red ?

thanks,

Joël

**Comment by jbaudoux** [comment]
Hello Joel,
I guess this is because the code was initially for version 7 and can't work in 8 without the changes marked by red cross (Travis failed). After those changes, code seems to work with 8 as you don't have the red cross any more.

**Comment by eLBati** [comment]
Thanks @jbaudoux 
some PEP8 to fix

```
./account_invoice_line_default_account/__openerp__.py:41:80: E501 line too long (80 > 79 characters)
./account_invoice_line_default_account/model/account_invoice_line.py:28:17: E128 continuation line under-indented for visual indent
./account_invoice_line_default_account/model/account_invoice_line.py:32:17: E128 continuation line under-indented for visual indent
./account_invoice_line_default_account/model/account_invoice_line.py:49:17: E128 continuation line under-indented for visual indent
./account_invoice_line_default_account/model/account_invoice_line.py:54:15: E128 continuation line under-indented for visual indent
./account_invoice_line_default_account/model/account_invoice_line.py:59:15: E128 continuation line under-indented for visual indent
./account_invoice_line_default_account/model/account_invoice_line.py:66:17: E126 continuation line over-indented for hanging indent
```

Also, you may want to rebase, to get a more recent .travis.yml

File: account_invoice_line_default_account/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': 'Account Invoice Line Default Account',
+    'version': '7.0.1.0',
+    'depends': [
+        'account'
+    ],
+    'author': 'Therp BV',
```
Comment (adrienpeiffer) [inline account_invoice_line_default_account/__openerp__.py]: Can you add OCA as author like this : Therp BV,Odoo Community Association (OCA)

File: account_invoice_line_default_account/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    'contributors': ['Jacques-Etienne Baudoux <je@bcim.be>'],
```
Comment (adrienpeiffer) [inline account_invoice_line_default_account/__openerp__.py:29]: Can you extract this into README.rst file

File: account_invoice_line_default_account/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    'category': 'Accounting',
+    'description': '''When entering sales or purchase invoices directly, the
```
Comment (adrienpeiffer) [inline account_invoice_line_default_account/__openerp__.py]: Can you extract this into README.rst file

File: account_invoice_line_default_account/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': 'Account Invoice Line Default Account',
+    'version': '7.0.1.0',
```
Comment (adrienpeiffer) [inline account_invoice_line_default_account/__openerp__.py]: I think 1.0 is enought

**Comment by jbaudoux** [comment]
Hello,
I update this PR with your comments. Hope it can get merged.
kr
J-E

File: account_invoice_line_default_account/README.rst [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+partner as default expense or income account, unless explicitly disabled in the
+partner record.
+
+Credits
+=======
+
+Contributors
+------------
+
+* Jacques-Etienne Baudoux <je@bcim.be> (BCIM sprl)
```
Comment (sbidoul) [inline account_invoice_line_default_account/README.rst:59]: Add original author in contributors.

**Comment by sbidoul** [comment]
Can you just update the README according to the OCA template in https://github.com/OCA/maintainer-tools/tree/master/template/module.

Otherwise :+1:

**Comment by jbaudoux** [comment]
Directory structure and README has been updated

**Comment by max3903** [comment]
:+1:

**Comment by adrienpeiffer** [comment]
@jbaudoux Can you check why travis is red?

**Comment by jbaudoux** [comment]
It's not related to this module. It's tests in account_invoice_line_sort module that are failing.

**Comment by jbaudoux** [comment]
Hello,
Can this old PR get merged?

**Comment by pedrobaeza** [comment]
Travis is still red. @jbaudoux, are you going to work on this?

**Comment by jbaudoux** [comment]
I think Travis will always be red unless the module account_invoice_line_sort in the branch 8.0 get fixed.

File: account_invoice_line_default_account/__openerp__.py [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': 'Account Invoice Line Default Account',
+    'version': '1.0',
```
Comment (pedrobaeza) [inline account_invoice_line_default_account/__openerp__.py:24]: Use 8.0.1.0.0

File: account_invoice_line_default_account/i18n/account_invoice_line_default_account.pot [hunk 7]
```diff
@@ -0,0 +1,59 @@
+# Translation of OpenERP Server.
```
Comment (pedrobaeza) [inline account_invoice_line_default_account/i18n/account_invoice_line_default_account.pot:1]: Remove POT file

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by pedrobaeza** [comment]
Superseeded by #189

**Pull request closed without merging**